ngraph/test/onnx/onnx_import.in.cpp

```diff
@@ -3199,7 +3199,7 @@ NGRAPH_TEST(${BACKEND_NAME}, onnx_group_norm)
 {
     const auto function = onnx_import::import_onnx_model(
         file_util::path_join(SERIALIZED_ZOO, "onnx/group_norm.prototxt"));
-    auto test_case = test::TestCase<TestEngine>(function);
+    auto test_case = test::TestCase<TestEngine, test::TestCaseType::DYNAMIC>(function);
```

Comment:
Should we add new tests or this change already allows to test changes?

Reply:
>  this change already allows to test changes

yes, adding DYNAMIC allows to test the changes to GroupNorm op.